src/OpenLoco/S5/S5.cpp

```diff
@@ -453,33 +453,20 @@ namespace OpenLoco::S5
         // Read packed objects
         if (file->header.numPackedObjects > 0)
         {
-            bool objectInstalled = false;
             for (auto i = 0; i < file->header.numPackedObjects; ++i)
             {
                 ObjectHeader object;
                 fs.read(&object, sizeof(ObjectHeader));
-                if (ObjectManager::tryInstallObject(object, fs.readChunk()))
-                {
-                    objectInstalled = true;
-                }
-            }
-
-            if (objectInstalled)
-            {
-                ObjectManager::loadIndex();
+                auto unownedObjectData = fs.readChunk();
+                std::vector<uint8_t> objectData;
```

Comment:
Might be nice to use `std::byte` here?
```suggestion
                std::vector<std::byte> objectData;
```

Reply:
I can look into that in a separate PR. Would be slightly messy as it needs changes in a few places.